backbone/src/main/java/org/researchstack/backbone/storage/database/sqlite/DatabaseHelper.java

```diff
@@ -82,7 +82,7 @@ public void saveTaskResult(TaskResult taskResult)
             taskRecord.completed = taskResult.getEndDate();
             getDao(TaskRecord.class).create(taskRecord);
 
-            Gson gson = new GsonBuilder().setDateFormat(FormatHelper.DATE_FORMAT_ISO_8601).create();
```

Comment:
it's how dates are formatted in ResearchKit. Anyway I had to revert back to the original format as this one was not supported well in Android unforunately.